src/Tizen.Network.Stc/Tizen.Network.Stc/StcInfo.cs

```diff
@@ -0,0 +1,324 @@
+            get
+            {
+                long incoming;
+                long outgoing;
+                int ret = Interop.Stc.Info.GetCounter(_infoHandle, out incoming, out outgoing);
+                if (ret != (int)StcError.None)
+                {
+                    Log.Error(Globals.LogTag, "Failed to get outgoing counter from info, Error - " + (StcError)ret);
+                }
+                return outgoing;
+            }
+        }
+
+        /// <summary>
+        /// A property to get the roaming type from statistics information.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// <value>IsRoaming.</value>
+        /// <privilege>http://tizen.org/privilege/network.get</privilege>
+        /// <exception cref="NotSupportedException">Thrown while setting this property when Stc is not supported.</exception>
+        /// <exception cref="InvalidOperationException">Thrown while setting this value due to an invalid operation.</exception>
+        public bool IsRoaming
+        {
+            get
+            {
+                RoamingType roaming;
+                int ret = Interop.Stc.Info.GetRoaming(_infoHandle, out roaming);
+                if (ret != (int)StcError.None)
+                {
+                    Log.Error(Globals.LogTag, "Failed to get Roaming type from info, Error - " + (StcError)ret);
+                }
+
+                if(roaming == RoamingType.Disabled)
+                {
+                    return false;
+                }
+                return true;
+            }
+        }
+
+        /// <summary>
+        /// A property to get the network protocol type from statistics information.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// <value>NetworkProtocol type.</value>
+        /// <privilege>http://tizen.org/privilege/network.get</privilege>
+        /// <exception cref="NotSupportedException">Thrown while setting this property when Stc is not supported.</exception>
+        /// <exception cref="InvalidOperationException">Thrown while setting this value due to an invalid operation.</exception>
+        public NetworkProtocol Protocol
+        {
+            get
+            {
+                NativeNetworkProtocol protocol;
+                int ret = Interop.Stc.Info.GetProtocol(_infoHandle, out protocol);
+                if (ret != (int)StcError.None)
+                {
+                    Log.Error(Globals.LogTag, "Failed to get Protocol type from info, Error - " + (StcError)ret);
+                }
+                if (protocol == NativeNetworkProtocol.Unknown)
+                {
+                    throw new InvalidOperationException("Network protocol is Unknown.");
+                }
+                else
+                {
+                    return (NetworkProtocol)protocol;
+                }
+            }
+        }
+
+        /// <summary>
+        /// A property to get the process state from statistics information.
```

Comment:
Could you explain more details?
I can't understand your description.

Reply:
Done.